modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -285,6 +285,12 @@ In {productname} 7.0, this problem has been addressed.
 
 As a result, the table ghost element now accurately mirrors adjustments in table height when resized using the corner resize handles.
 
+=== Accurate default options in the `styles` bespoke select toolbar button
```

Comment:
Should the release note entry directly reflect [the changelog](https://github.com/tinymce/tinymce/blob/77f9b167e416ecbb4fbe58f6280780792e646d31/.changes/unreleased/tinymce-TINY-10603-2024-02-26.yaml)?